docusaurus/docs/iOS/guides/video-integration.md

```diff
@@ -0,0 +1,300 @@
+
+In the provided repository, you'll find a [`StreamWrapper`](https://github.com/GetStream/stream-video-ios-examples/blob/main/ChatWithVideo/ChatWithVideo/StreamWrapper.swift) that facilitates the initialization and connection of both clients.
+
+Here's an example of how to set up the `StreamWrapper`:
+
+```swift
+let streamWrapper = StreamWrapper(
+    apiKey: "YOUR_API_KEY",
+    userCredentials: user,
+    tokenProvider: { result in
+        let token = \\ Retrieve the token from your backend
+        result(.success(token))
+    }
+)
+```
+
+Keep in mind that there are alternative ways to configure the clients without using this wrapper. Additional details on setting up the Stream Video SDK can be found [here](https://getstream.io/video/docs/ios/basics/quickstart/).
+
+### Adding a UI element
+
+The simplest way to add video calling in a chat app is to customize the chat channel header and add a call icon in the top right corner.
+
+To do this, you need to implement the `makeChannelHeaderViewModifier` function in your custom implementation of the `ViewFactory` from the Stream Chat SDK:
+
+```swift
+@MainActor
+func makeChannelHeaderViewModifier(for channel: ChatChannel) -> some ChatChannelHeaderViewModifier {
+    CallHeaderModifier(channel: channel, callViewModel: callViewModel)
+}
+```
+
+For this code to compile you need a `CallHeaderModifier` that looks like this:
+
+```swift
+struct CallHeaderModifier: ChatChannelHeaderViewModifier {
+
+    var channel: ChatChannel
+    var callViewModel: CallViewModel
+
+    func body(content: Content) -> some View {
+        content.toolbar {
+            CallChatChannelHeader(channel: channel, callViewModel: callViewModel)
+        }
+    }
+
+}
+```
+
+This creates a toolbar with the content of type `CallChatChannelHeader` that can be created like this (this is simplified code, find the [full version here](https://github.com/GetStream/stream-video-ios-examples/blob/main/ChatWithVideo/ChatWithVideo/ChatViewFactory.swift)):
+
+```swift
+public struct CallChatChannelHeader: ToolbarContent {
+
+    private var shouldShowTypingIndicator: Bool { /* ... */ }
+
+    public var channel: ChatChannel
+    @ObservedObject var callViewModel: CallViewModel
+
+    public var body: some ToolbarContent {
+        // highlight-next-line
+        // 1. Add the title of the channel in the center position
+        ToolbarItem(placement: .principal) {
+            ChannelTitleView(
+                channel: channel,
+                shouldShowTypingIndicator: shouldShowTypingIndicator
+            )
+        }
+        // highlight-next-line
+        // 2. Add a trailing item to the toolbar with a phone icon
+        ToolbarItem(placement: .navigationBarTrailing) {
+            Button {
+                // highlight-next-line
+                // 3. Create and start a call with the participants from the channel
+                let participants = channel.lastActiveMembers.map { member in
```

Comment:
Is `lastActiveMembers` the right way to get the channel members?

Reply:
Yes, it's fine, since we don't want to paginate and call more than 100 people at once. Direct calling is meant for smaller groups, and last active members returns that.